numba/tests/test_range.py

```diff
@@ -63,6 +63,16 @@ def range_attrs(start, stop, step):
         tmp.append((r.start, r.stop, r.step))
     return tmp
 
+def range_contains(val, start, stop, step):
+    r1 = range(start)
+    r2 = range(start, stop)
+    r3 = range(start, stop, step)
+    tmp = []
```

Comment:
It's more pythonic to return a list comprehension here.

```
return [val in r for r in (r1, r2, r3)]
```

Reply:
done